pkgs/development/python-modules/pykeepass/default.nix

```diff
@@ -15,6 +16,7 @@ buildPythonPackage rec {
   propagatedBuildInputs = [
     lxml pycryptodome construct
     argon2_cffi dateutil enum34
```

Comment:
`enum34` seems to have been removed

Reply:
One would think so, since it is not in the specified dependencies, but it is still required to build this package, this is the output with enum34 removed:

```
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7.
Processing ./pykeepass-3.0.3-py2-none-any.whl
Requirement already satisfied: lxml in /nix/store/8mhziwf7vpmnk013zqs68jhmpk58iih5-python2.7-lxml-4.3.1/lib/python2.7/site-packages (from pykeepass==3.0.3) (4.3.1)
Requirement already satisfied: construct in /nix/store/pv8pyazg28w9d2p4snfq4mwf3kzg0q0z-python2.7-construct-2.9.45/lib/python2.7/site-packages (from pykeepass==3.0.3) (2.9.45)
Requirement already satisfied: pycryptodome in /nix/store/pz0b5hrdbgnfcfyvdr065vfsnj48cllz-python2.7-pycryptodome-3.7.3/lib/python2.7/site-packages (from pykeepass==3.0.3) (3.7.3)
Requirement already satisfied: argon2-cffi in /nix/store/q1jhvxiw14byyzfymb08slln2hklm38y-python2.7-argon2_cffi-19.1.0/lib/python2.7/site-packages (from pykeepass==3.0.3) (19.1.0)
Requirement already satisfied: future in /nix/store/dpa5hy4shh8qknkkr83mb5r6my7cfnaf-python2.7-future-0.17.1/lib/python2.7/site-packages (from pykeepass==3.0.3) (0.17.1)
Requirement already satisfied: python-dateutil in /nix/store/lpdclq0k00v6npdzl0hrdfxavr506n3b-python2.7-python-dateutil-2.8.0/lib/python2.7/site-packages (from pykeepass==3.0.3) (2.8.0)
Requirement already satisfied: six in /nix/store/2i40ny9bh5rxismw58kz723j9lh64fxr-python2.7-six-1.12.0/lib/python2.7/site-packages (from argon2-cffi->pykeepass==3.0.3) (1.12.0)
Requirement already satisfied: cffi>=1.0.0 in /nix/store/43vsgf68g8knf6886vxjfcm5irqwbs34-python2.7-cffi-1.12.1/lib/python2.7/site-packages (from argon2-cffi->pykeepass==3.0.3) (1.12.1)
Collecting enum34 (from argon2-cffi->pykeepass==3.0.3)
  Could not find a version that satisfies the requirement enum34 (from argon2-cffi->pykeepass==3.0.3) (from versions: )
No matching distribution found for enum34 (from argon2-cffi->pykeepass==3.0.3)
builder for '/nix/store/6q0k9kmvcdh0l6a9f5nr8qw89pf0hd0l-python2.7-pykeepass-3.0.3.drv' failed with exit code 1
error: build of '/nix/store/6q0k9kmvcdh0l6a9f5nr8qw89pf0hd0l-python2.7-pykeepass-3.0.3.drv' failed
```

Seems it is an unspecified dependency of `argon2_cffi`.

Thanks for catching this, I will update with a new commit.